package.json

```diff
@@ -52,7 +52,7 @@
     "release": "ts-node --transpile-only -P automation/tsconfig.json automation/run.ts release",
     "pretest": "npm run build",
     "test": "mocha --timeout 6000 -r ts-node/register/transpile-only \"tests/**/*.spec.ts\"",
-    "test:fast": "npm run build:fast && npm run test",
+    "test:fast": "npm run build:fast && mocha --timeout 6000 -r ts-node/register/transpile-only \"tests/**/*.spec.ts\"",
```

Comment:
This modification is in its own commit because it is an independent fix. What happens is that `npm test` triggers the `pretest` script that executes `npm run build`. Before this change, `npm run test:fast` would run `npm run build:fast` but then also run `npm run build` (because of `prestest`), therefore building twice and being slower than `npm test`.